core/designsystem/src/main/java/com/acon/core/designsystem/component/button/AconGoogleLoginButton.kt

```diff
@@ -0,0 +1,76 @@
+package com.acon.core.designsystem.component.button
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.acon.core.designsystem.R
+import com.acon.core.designsystem.theme.AconTheme
+
+@Composable
+fun AconGoogleLoginButton(
+    backgroundColor: Color = AconTheme.color.White,
+    borderColor: Color = AconTheme.color.Gray1,
+    modifier: Modifier = Modifier,
+    borderWidth: Dp = 1.dp,
+    cornerRadius: Dp = 6.dp,
+    textStyle: TextStyle = AconTheme.typography.subtitle2_14_med,
+    textColor: Color = AconTheme.color.Black,
+    onClick: () -> Unit = {}
+) {
+    AconButton(
+        backGroundColor = backgroundColor,
+        borderColor = borderColor,
+        modifier = modifier
+            .fillMaxWidth(),
+        borderWidth = borderWidth,
+        cornerRadius = cornerRadius,
+        paddingHorizontal = 93.dp,
+        paddingVertical = 12.dp,
+        onClick = onClick
+    ) {
+        Row(
+           verticalAlignment = Alignment.CenterVertically
+        ) {
+            Image(
+                painter = painterResource(R.drawable.ic_google),
```

Comment:
```suggestion
                imageVector = ImageVector.vectorResource(R.drawable.ic_google)
```
ImageVector는 이렇게 사용해주는 것이 리컴포지션을 줄일 수 있습니당